packages/govuk-frontend/src/govuk/components/tabs/tabs.mjs

```diff
@@ -94,6 +91,7 @@ export class Tabs extends GOVUKFrontendComponent {
 
     this.$tabList = $tabList
     this.$tabListItems = $tabListItems
+    this.mql = window.matchMedia('(min-width: 40.0625em)')
```

Comment:
**question** What's prompting the move of that line out of `setupResponsiveChecks`, the method was neatly encapsulating all the responsive setup before?

Reply:
The compiler probably thinks that `!$tabListItems.length` causes it to stay as `mql = null`

By moving it into the constructor we don't need to add extra `null` checks in `setupResponsiveChecks()`

Reply:
Moved to https://github.com/alphagov/govuk-frontend/pull/4286